src/views/Contribute/Contribute.tsx

```diff
@@ -0,0 +1,311 @@
+import {
+  Accordion,
+  Stack,
+  OrderedList,
+  ListItem,
+  Text,
+  UnorderedList,
+} from "@chakra-ui/react";
+import type { FunctionComponent } from "react";
+import { FAQ, FAQHeading, FAQItem, FAQLink } from "../../components/FAQ";
+import { Page } from "../../components/Page";
+
+export const Contribute: FunctionComponent = () => (
+  <Page
+    meta={{
+      description: "Learn about contributing to Construct Hub.",
+      title: "Contribute",
+    }}
+    pageName="contribute"
+  >
+    <FAQ>
+      <FAQHeading>Contributing to Construct Hub</FAQHeading>
+
+      <Accordion allowMultiple allowToggle defaultIndex={[0]}>
+        <FAQItem question="How can I add my construct to Construct Hub?">
+          <Stack spacing={4}>
+            <Text>
+              Constructs intended for Construct Hub must be published to the{" "}
+              <FAQLink href="https://www.npmjs.com/">NPM registry</FAQLink>{" "}
+              under a permissive license (such as Apache, BSD, EPL, MPL-2.0, ISC
+              and CDDL or MIT) and annotated with a keyword recognized by
+              Construct Hub (awscdk, cdk8s, or cdktf).
```

Comment:
It looks like this question is the same as one in the FAQ, but the answer text doesn't match (the list of licenses is different, and it's missing a link on "file an issue against our GitHub repository"). The text should be fixed to match (or we could remove the question from the FAQ and only keep the copy here).